components/x-teaser/src/Image.jsx

```diff
@@ -17,20 +17,24 @@ const aspectRatio = ({ width, height }) => {
 	return null;
 };
 
-const NormalImage = ({ src }) => (
-	<img className="o-teaser__image" src={src} alt="" />
+const NormalImage = ({ src, srcSet }) => (
+	<img className="o-teaser__image" src={src} srcSet={srcSet} alt="" />
```

Comment:
If `srcSet` is null then the attribute will be empty.

```js
NormalImage({ src: 'foo', srcSet: null }) // `<img className="o-teaser__image" src="foo" srcSet="" alt="" />
```

I think it shouldn't be set if the value is falsey.